src/Components/Home/Home.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+//Components
+import Header from '../Header/Header';
+
+class Home extends Component {
+    state = {
```

Comment:
Why did you create a state here? If it is not necessary yet, please remove it and create the component as a function component instead of class component.